.env

```diff
@@ -1,3 +1,6 @@
 # NOTE: This is needed forabsolute path imports to work
 # https://itnext.io/configure-absolute-paths-with-create-react-app-and-flow-e4b8922676a2
 NODE_PATH=src/
+
+NODE_PUBLIC_URL=https://72aa6e93.ngrok.io
+REACT_APP_PUBLIC_URL=https://72aa6e93.ngrok.io
```

Comment:
:bomb: or comment out 
This will probably break production, staging and PR builds since these are injected by Heroku through deployment config